pkg/karmadactl/cmdinit/cert/cert.go

```diff
@@ -261,11 +262,12 @@ func NewCertConfig(cn string, org []string, altNames certutil.AltNames, notAfter
 }
 
 // GenCerts Create CA certificate and sign etcd karmada certificate.
-func GenCerts(pkiPath string, etcdServerCertCfg, etcdClientCertCfg, karmadaCertCfg, apiserverCertCfg, frontProxyClientCertCfg *CertsConfig) error {
-	caCert, caKey, err := NewCACertAndKey("karmada")
+func GenCerts(pkiPath, caCertPath, caKeyPath string, etcdServerCertCfg, etcdClientCertCfg, karmadaCertCfg, apiserverCertCfg, frontProxyClientCertCfg *CertsConfig) error {
+	caCert, caKey, err := getCertAndKey(caCertPath, caKeyPath)
 	if err != nil {
 		return err
 	}
+
 	if err = WriteCertAndKey(pkiPath, globaloptions.CaCertAndKeyName, caCert, caKey); err != nil {
 		return err
 	}
```

Comment:
if you provided ca/key file, why should we [read it from file -> parse it -> write back to file], why not just move it to `pkiPath`?

Reply:
1：if parse error, we can return the error in init.
2：less code logic change， keep the  return values the same as  with `NewCACertAndKey("karmada")`.

Reply:
> 1：if parse error, we can return the error in init.

it make sense